backend/config/corpora-api.yml

```diff
@@ -413,6 +413,39 @@ paths:
         "404":
           $ref: "#/components/responses/404"
 
+  /dp/v1/collections/index:
+    get:
+      tags:
+        - collections
+      summary: >-
+        Returns all public, non tombstoned collections. This endpoint is meant to be used for displaying collections
+        in a compact view. It doesn't return all the fields, so it is not suitable for general usage.
+      security:
```

Comment:
This can be removed if the functionality is the name for both authenticated and unauthenticated.